eclipse-formatter.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="1">
+<profile kind="CodeFormatterProfile" name="ApacheKafka" version="1">
+    <!-- Import ordering -->
+    <setting id="org.eclipse.jdt.core.formatter.blank_lines_between_import_groups" value="1"/>
+    <setting id="org.eclipse.jdt.core.formatter.blank_lines_before_imports" value="1"/>
+    <setting id="org.eclipse.jdt.core.formatter.blank_lines_after_imports" value="1"/>
+
+    <!-- Line -->
+    <setting id="org.eclipse.jdt.core.formatter.lineSplit" value="120"/>
+    <setting id="org.eclipse.jdt.core.formatter.comment.line_length" value="120"/>
```

Comment:
http://kafka.apache.org/coding-guide.html says:

"There is not a maximum line length (certainly not 80 characters, we don't work on punch cards any more), but be reasonable."

In Streams, we prefer a maximum line length of 120 but we do not enforce it with checkstyle (at least I could not find a rule for it). While I do not like long lines, I also think that sometimes there are good reasons to exceed the limit. So I would not enforce it in the formatter.

Reply:
Agree. It seems like removing this rule would be better.